docs/github-action.md

```diff
@@ -0,0 +1,34 @@
+# Copa Github Action
```

Comment:
should we move this to website so it's more visible? wdyt?

Reply:
yes! moved

Reply:
please make sure to update sidebar